### PR TITLE
IGNITE-15049: Update the documentation for the cluster performance pr…

### PR DESCRIPTION
**Pull request by Sega76**
…ofiling tool.

Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [x] There is a single JIRA ticket related to the pull request. 
- [x] The web-link to the pull request is attached to the JIRA ticket.
- [x] The JIRA ticket has the _Patch Available_ state.
- [x] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [x] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: docs/_docs/monitoring-metrics/performance-statistics.adoc [hunk 1]
```diff
@@ -80,9 +80,41 @@ You can manage the performance statistics collection via the `PerformanceStatist
 |Method | Description
 |start() | Start collecting performance statistics in the cluster.
 |stop() | Stop collecting performance statistics in the cluster.
+|rotate() | Rotate collecting performance statistics in the cluster.
 |started() | True if performance statistics collection is started.
 |===
 
+
+== Control Script
```
Comment (NSAmelchev) [inline docs/_docs/monitoring-metrics/performance-statistics.adoc:88]: Lets use level 2 section title

File: docs/_docs/tools/control-script.adoc [hunk 2]
```diff
@@ -913,3 +915,35 @@ TRANSACTIONS                   SYS       Running transactions
 CACHES                         SYS       Caches
 TASKS                          SYS       Running compute tasks
 Command [SYSTEM-VIEW] finished with code: 0
+
+
+== Performance Statistics
+
+Ignite provides a built-in tool for cluster profiling.
```
Comment (NSAmelchev) [inline docs/_docs/tools/control-script.adoc:922]: Lets add link to page: `Read link:monitoring-metrics/performance-statistics[Performance Statistics] for more information.`

File: docs/_docs/monitoring-metrics/performance-statistics.adoc [hunk 3]
```diff
@@ -80,9 +80,41 @@ You can manage the performance statistics collection via the `PerformanceStatist
...
+
+You can manage the performance statistics collection via the command line interface.
```
Comment (NSAmelchev) [inline docs/_docs/monitoring-metrics/performance-statistics.adoc:90]: Lets add link to the Control script page

File: docs/_docs/monitoring-metrics/performance-statistics.adoc (hunk 1 above)
Comment (Sega76) [reply docs/_docs/monitoring-metrics/performance-statistics.adoc]: done

File: docs/_docs/monitoring-metrics/performance-statistics.adoc (hunk 3 above)
Comment (Sega76) [reply docs/_docs/monitoring-metrics/performance-statistics.adoc]: done

File: docs/_docs/tools/control-script.adoc (hunk 2 above)
Comment (Sega76) [reply docs/_docs/tools/control-script.adoc]: done

**Review by NSAmelchev — APPROVED** [review]

**Pull request merged**